studies/models.py

```diff
@@ -512,7 +512,7 @@ def notify_administrators_of_submission(self, ev):
                     name=SiteAdminGroup.LOOKIT_ADMIN.name
                 ).user_set.values_list("username", flat=True)
             ),
-            reply_to=[self.request.user.username],
+            reply_to=[ev.kwargs.get("user")],
```

Comment:
this is actually passing a User object rather than just the username (email address) - I think you should just be able to add `.username` here though!

Reply:
This should be resolved by commit https://github.com/lookit/lookit-api/pull/671/commits/a60970f34d7c3c746a6d2abf5bfc7aa9cd551d43.